squiRL/common/agents.py

```diff
@@ -66,8 +67,8 @@ def get_action(
         action_logit = net(obs)
         probs = F.softmax(action_logit, dim=-1)
         dist = torch.distributions.Categorical(probs)
-        action = dist.sample().item()
-        action = int(action)
+        action = dist.sample().squeeze().numpy()
+        action = np.expand_dims(action, axis=0) if action.ndim == 0 else action
```

Comment:
Why on line 70 is there a squeeze and then an unsqueeze on line 71? I can see the second line is conditional and probably to do with whether it is a single state or a batch but seems like reshaping to [1,-1] might be better?

Reply:
`gym3` crashes when the action shape takes the form `(1, n_envs)`. It needs to pass as `(n_envs)`. In the single env case, line 70 returns shape `()`. Just an `int`, hence we expand its dims to be `(1,)`. In all other cases, line 70 outputs the correct shape of `(n_env)`.

Reply:
Nice, thanks for the clarification.